core/src/main/java/org/bitcoinj/base/internal/ByteUtils.java

```diff
@@ -289,6 +289,19 @@ public static void writeUint32LE(long val, OutputStream stream) throws IOExcepti
         stream.write(buf);
     }
 
+    /**
+     * Write a 4-byte integer to the output stream in little endian format.
+     * The integer may be signed or unsigned, its bytes are just serialized as is.
+     * @param val value to be written
+     * @param stream stream to be written into
+     * @throws IOException if an I/O error occurs
+     */
+    public static void writeInt32LE(int val, OutputStream stream) throws IOException {
```

Comment:
I think this write helper can be removed, as per our agreement to not have unused helpers methods.